tutorials/optimviz/WeightVisualization_OptimViz.ipynb

```diff
@@ -821,7 +821,7 @@
         "        optimviz.transform.RandomScale(scale=(1, 0.975, 1.025, 0.95, 1.05)),\r\n",
         "        torchvision.transforms.RandomRotation(degrees=(-5,5)),\r\n",
         "        optimviz.transform.RandomSpatialJitter(8),\r\n",
-        "        optimviz.transform.CenterCrop(16),\r\n",
+        "        optimviz.transform.CenterCrop((224,224)),\r\n",
```

Comment:
question: Would this be equivalent to passing: optimviz.transform.CenterCrop(16, pixels_from_edges=True) ?

Reply:
Yeah, it should be equivalent, but one of the transforms subtracts two rows of values from the H or W dimensions. I ran into that issue while trying to put together a neural style transfer tutorial (likely to be combined with some other small tutorials). It's not a major issue, but I'm working on a fix that will likely be included in a future PR.

Reply:
Okay, so there isn't a bug! For removing the padding from `torch.nn.ReflectionPad2d(16)`, the equivalent to  `CenterCrop(224, False)`, is `CenterCrop(32, True)`.

Reply:
I see, so was the original line `optimviz.transform.CenterCrop(16)` incorrect ? because it looks like the original cropped 8 from each side ?

Reply:
@NarineK Yeah, it was! I mistakenly put 16 as I forgot about the division by 2!

Reply:
I can change it so that 16 is the correct user input if you think it would be better.